protocol/avdl/keybase1/account.avdl

```diff
@@ -76,4 +76,22 @@ protocol account {
    */
   void cancelReset(int sessionID);
   void timeTravelReset(int sessionID, string username, gregor1.DurationSec duration);
+
+  record TeamContactSettings {
+    TeamID teamID;
+    boolean allowFolloweesOfTeamMembers;
+    boolean enabled;
+  }
+
+  record ContactSettings {
+    int version;
+    boolean allowFolloweeDegrees;
+    boolean enabled;
+    array<TeamContactSettings> teams;
+  }
+  
+  ContactSettings userGetContactSettings(UID uid);
+
+  void userSetContactSettings(boolean allowFolloweeDegrees, boolean enabled, array<TeamContactSettings> teams);
```

Comment:
can you just pass `ContactSettings settings` here? Server will just ignore `version`, which is fine.

Reply:
looks like then it will make it such that in service/account.go, that i have to do `args.Settings` to get the args

Reply:
I think it'll work given the suggestions I made inline...